certbot/certbot/_internal/cli/__init__.py

```diff
@@ -321,12 +321,12 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):
         "--redirect", action="store_true", dest="redirect",
         default=flag_default("redirect"),
         help="Automatically redirect all HTTP traffic to HTTPS for the newly "
-             "authenticated vhost. (default: Ask)")
+             "authenticated vhost. (default: False for enhance, True for renew and run)")
     helpful.add(
         "security", "--no-redirect", action="store_false", dest="redirect",
         default=flag_default("redirect"),
         help="Do not automatically redirect all HTTP traffic to HTTPS for the newly "
-             "authenticated vhost. (default: Ask)")
+             "authenticated vhost. (default: False for enhance, True for renew and run)")
```

Comment:
I decided not to remove these from `security` because they show up next to each other during `all` so I wanted to find a way that makes sense even if they're next to each other. I considered adding this one under `redirect` and `run` but honestly think we have enough flags there and we didn't need it there before.

Reply:
yes, I love that, stateful booleans are inherently more confusing than idempotent statements.